roles/backup-server/templates/usr/local/bin/run-backups.sh.j2

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
```

Comment:
Awesome! Now this is something we could monitor with Datadog, either the log file living way too long, or otherwise lack of backups happening. Thanks!